### DISCUSSION TITLE
Calls dedicated Element UX setting

### DISCUSSION THREAD

**Discussion by Thatoo** [element-hq/element-meta#852] [Ideas]
#### What would you like to do?
Use Element (matrix) only for calls.

#### Why would you like to do it?
- Make calls usage more accessible. 
- Some people don't want to embarrass themselves with chat system but would like to use matrix to call each other (elderly in family)
- Intercom within buildings, events, company...
#### How would you like to achieve it?
Add a theme option in the settings that would be dedicated to calls only 

Only a navigation system to select a room and once we select a room, the button to start or join a call (in case of a group Element Call).
Being able to answer a call would be required and that's all, no text chat and no advance usage.

### Have you considered any alternatives?

I wish we could have a lightweight matrix client such as Hydrogen dedicated to calls only.

**Comment by Thatoo** [comment]
Here is a mock-up how it could look like

To call someone : 

![elementcallclient-2](https://user-images.githubusercontent.com/464782/209203481-340fc2b5-cb92-47fd-8fb5-b21031c7dee4.png)

And during the call : 

![elementcallclient-3](https://user-images.githubusercontent.com/464782/209203654-1928f28d-940c-45ee-96a7-291d7aedea41.png)

Very simple, very light UX design, only the list of users and list of rooms that have a group video call widget enabled on the left and, on the right, button to start a call and button needed during a call.

Of course it means that one should use an other client to add users and join rooms or start rooms and add video call widget to a room.

**Comment by SimonBrandner** [comment]
We already have call.element.io, does that satisfy your needs?

**Reply by Thatoo** [reply]
That was my first idea actually but then, when I read requested features in the github repo of Element Call, I found my request but it has been closed with a no and here is the answer https://github.com/vector-im/element-call/issues/722#issuecomment-1308888270

Apparently Element Call can't have the list of users and rooms so I it can't answer this particular need.

**Comment by Thatoo** [comment]
I wonder if Hydrogen would not be more appropriate to do the job then.
I open a issue there https://github.com/vector-im/hydrogen-web/issues/943
